libraries/stdlib/src/kotlin/io/IOStreams.kt

```diff
@@ -13,39 +13,45 @@ public fun InputStream.iterator(): ByteIterator =
             public override fun nextByte(): Byte = read().toByte()
         }
 
-/** Creates a buffered input stream */
+/**
+ * Creates a buffered input stream wrapping this stream.
+ * @param bufferSize the buffer size to use.
+ */
 public fun InputStream.buffered(bufferSize: Int = defaultBufferSize): InputStream
         = if (this is BufferedInputStream)
     this
 else
     BufferedInputStream(this, bufferSize)
 
-/** Creates a reader on an input stream using UTF-8 or specified charset. */
+/** Creates a reader on this input stream using UTF-8 or the specified [charset]. */
```

Comment:
Need to put it in conventions, wording around documenting parameter with default value.